src/master/production/sync-replication.md

```diff
@@ -192,21 +192,205 @@ Make sure you already have [ImmuDB installed](../running/download.md).
    Login to immudb
 
    ```shell
-   $ ./immuclient login immudb
+   $ immuclient login immudb
    ```
 
    Select database
 
    ```shell
-   $ ./immuclient use primarydb
+   $ immuclient use primarydb
    ```
 
    Set a value
 
    ```shell
-   $ ./immuclient safeset foo bar
+   $ immuclient safeset foo bar
    ```
 
    The client will block. This is because the primarydb requires 1 sync follower, and since the replica server is down, there is no ack from the replica server, hence synchronous transaction is blocked.
 
 </WrappedSection>
+
+<WrappedSection>
+
+## Recovering from a replica loss
+
+The primary node will continue read and write operations as long as enough replicas can send a write confirmations to the primary node.
```

Comment:
wdyt of using the quorum word instead of enough replicas? Because since we have defined it above in the doc that we use a quorum based technique, maybe we could use `required quorum of replica server` or something similar

Reply:
Enough replica could lead to confusion on what is the definition of enough